### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by lafriks**
As per title

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,10 +1,12 @@
 module github.com/yeqown/fasthttp-reverse-proxy/v2
 
 require (
-	github.com/fasthttp/websocket v1.4.1
-	github.com/stretchr/testify v1.6.1 // indirect
-	github.com/valyala/fasthttp v1.4.0
-	github.com/yeqown/log v1.0.5
+	github.com/andybalholm/brotli v1.0.3 // indirect
```
Comment (yeqown) [inline go.mod:4]: remove indirect dependency

Comment (lafriks) [reply go.mod:4]: That's added by go mod so removing it might not be good idea

Comment (yeqown) [reply go.mod:4]: which version of Go are you using?

Comment (lafriks) [reply go.mod:4]: `go version go1.17.1 linux/amd64`

Comment (yeqown) [reply go.mod:4]: And I tried to upgrade the version of `fasthttp`, go mod would not add these two packages too ~

Comment (lafriks) [reply go.mod:4]: Hard to tell, did you run go mod tidy?

Comment (yeqown) [reply go.mod:4]: yeah

**Pull request merged**